Sources/Brave/Frontend/Browser/ThirdPartySearchAlerts.swift

```diff
@@ -16,19 +16,20 @@ class ThirdPartySearchAlerts: UIAlertController {
   }
 
   /**
-     Builds the Alert view that asks if the users wants to add a third party search engine.
+   Builds the Alert view that asks if the users wants to add a third party search engine.
 
-     - parameter engine: To add engine details to alert
-
-     - parameter completion: Okay option handler.
-
-     - returns: UIAlertController for asking the user to add a search engine
-     **/
+   - parameter engine: To add engine details to alert
+   - parameter completion: Okay option handler.
+   - returns: UIAlertController for asking the user to add a search engine
+  **/
 
   static func addThirdPartySearchEngine(_ engine: OpenSearchEngine, completion: @escaping (UIAlertAction) -> Void) -> UIAlertController {
     let alertMessage = """
       \n\(engine.displayName)
+      \nSearch Template:
       \(engine.searchTemplate)
+      \nSuggestion Template:
```

Comment:
Localization?

Reply:
Ohhh missed that fix at https://github.com/brave/brave-ios/pull/7721/commits/02941ed05e557a8831d25723423aba8263c62d2b